merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/GetSimulationTemplateAction.java

```diff
@@ -28,13 +27,11 @@ public GetSimulationTemplateAction(final Connection connection) throws SQLExcept
     this.statement = connection.prepareStatement(sql);
   }
 
-  public Optional<SimulationTemplateRecord> get(final long simulationTemplateId)
+  public SimulationTemplateRecord get(final long simulationTemplateId)
```

Comment:
If it's no longer safe to call this action on a non-existent simulation template id - can we add a javadoc comment documenting what it will do if it is provided a non-existent id? It is true that it is currently called in a safe fashion, but by changing the behavior from "returns Optional.empty()" to "throws an exception, probably?", we've made it more difficult for a future programmer to use this action safely.

Reply:
Resolving as I've dropped the commit this conversation is about.